airflow/contrib/executors/kubernetes_executor.py

```diff
@@ -458,10 +459,16 @@ def _datetime_to_label_safe_datestring(datetime_obj):
 
     def _labels_to_key(self, labels):
         try:
+            try_num = 1
```

Comment:
Can this block be moved before the try on 461? The try in the try doesn't read well.